templates/store/user-details.html

```diff
@@ -0,0 +1,110 @@
+{% extends "_layout.html" %}
+
+{% block title %}{{ context.username }}{% endblock %}
+
+{% block content %}
+
+<div class="p-strip--light is-bordered is-shallow" data-js="sticky-header">
```

Comment:
I think we can drop the sticky header now.

Reply:
Done.